src/Config.php

```diff
@@ -1302,6 +1305,21 @@ function formatGlobalLayout(layout) {
             echo "</td><td></td></tr>";
             echo "</td></tr>";
 
+            echo "<tr class='tab_bg_2'><td><label for='timeline_date_format$rand'>" . __('Timeline date display') .
+            "</label></td><td>";
+            if (!$userpref || Session::haveRight('ticket', Ticket::OWN)) {
```

Comment:
I think this check on `!$userpref || Session::haveRight('ticket', Ticket::OWN)` is not relevant here. Probably due to a copy/paste from `set_default_requester` option.
Anyone should be able to define its preferable timeline date format, and it should also be configurable globally.

`timeline_action_btn_layout` seems to suffer from same copy/paste side effect and should also be fixed.

Reply:
done
For ```timeline_action_btn_layout``` I will make a new PR